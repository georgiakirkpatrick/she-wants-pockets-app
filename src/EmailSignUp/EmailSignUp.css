.EmailSignUp {
    padding: 20px 10px;
}

.EmailSignUp > form > fieldset {
    position: relative;
    width: 100%;
    border: none;
    padding: 0 0 20px;
}

.EmailSignUp > form > fieldset > label {
    display: block;
    /* font-weight: 600; */
    padding: 10px 0 0;
}

.EmailSignUp > form > fieldset > input {
    background-color: rgb(255, 255, 255);
    border-left: none;
    border-top: none;
    border-right: none;
    border-bottom: solid 1px grey;
    padding: 5px;
    width: 100%;
    font-size: 16px;
}