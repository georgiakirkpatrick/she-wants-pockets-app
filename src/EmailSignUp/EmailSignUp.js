import React from 'react';
import './EmailSignUp.css';

function EmailSignUp() {
    return (
        <section>
            <header>
                <h3>
                    Save your favorite brands and styles
                </h3>
            </header>
            <p>
                Create an account and save items to buy later.
            </p>
            <form>
                <label htmlFor='email-signup'>Email address</label>
                <input type='text' name='email-signup' id='email-signup'></input>
                <button type='button'>Create account</button>
            </form>
        </section>
    )
};

export default EmailSignUp;