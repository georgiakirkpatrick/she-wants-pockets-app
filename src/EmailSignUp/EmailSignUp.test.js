import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import EmailSignUp from './EmailSignUp';

describe ('EmailSignUp', () => {
    it ('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<EmailSignUp />, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it ('renders the UI as expected', () => {
        const tree = renderer
            .create(<EmailSignUp />)
            .toJSON()
        expect(tree).toMatchSnapshot()    
    })
})