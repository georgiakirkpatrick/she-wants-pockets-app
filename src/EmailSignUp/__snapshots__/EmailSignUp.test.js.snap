// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailSignUp renders the UI as expected 1`] = `
<section>
  <header>
    <h3>
      Save your favorite brands and styles
    </h3>
  </header>
  <p>
    Create an account and save items to buy later.
  </p>
  <form>
    <label
      htmlFor="email-signup"
    >
      Email address
    </label>
    <input
      id="email-signup"
      name="email-signup"
      type="text"
    />
    <button
      type="button"
    >
      Create account
    </button>
  </form>
</section>
`;
